docs/api-reference/index.rst

```diff
@@ -5,4 +5,5 @@ API Reference
    :maxdepth: 2
 
    argument-parsing
-   module-helpers
```

Comment:
The helpers documentation wasn't being built properly because the name of the file was wrong. :/